app/src/main/java/com/example/dine_and_donate/HomeFragments/MapFragment.java

```diff
@@ -99,7 +100,7 @@
 
     private FirebaseUser currentUser;
     private FirebaseAuth mAuth;
-    private User currentUserModel;
+    private HomeActivity HomeActivity;
```

Comment:
Doesn't seem like this needs to be a field either? You can just do getActivity().mCurrentUser.isOrg;